assets/src/edit-story/components/devTools/devTools.js

```diff
@@ -0,0 +1,210 @@
+            newElement.resource.poster =
+              'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAGElEQVQYV2NkYGD4////fwZGEMnIyMgAAEvqB/6wfXLSAAAAAElFTkSuQmCC';
+
+            const videoEl = document.getElementById(`video-${newElement.id}`);
+            if (videoEl) {
+              videoEl.load();
+            }
+          }
+          if (element.type === 'image') {
+            newElement.resource.mimeType = 'image/png';
+            newElement.resource.src =
+              'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAGElEQVQYV2NkYGD4////fwZGEMnIyMgAAEvqB/6wfXLSAAAAAElFTkSuQmCC';
+          }
+        }
+        return newElement;
+      }),
+    })),
+  };
+};
+
+function DevTools() {
+  const [isVisible, setIsVisible] = useState(false);
+  const [isBase64, setIsBase64] = useState(false);
+  const [isDummyResources, setIsDummyResources] = useState(false);
+  const { showSnackbar } = useSnackbar();
+  const textareaRef = useRef();
+  const {
+    state: { reducerState },
+    actions: { restore },
+  } = useStory();
+  // This will not work here
```

Comment:
This would definitely not work here. It has a completely different context. We have to string things along to the `useStory()`, but we could provide it as `internal: {reducerState, restore}` to make it clear it's not for normal use.